src/components/OSCALLoader.js

```diff
@@ -192,12 +224,31 @@ export default function OSCALLoader(props) {
   if (!isLoaded) {
     result = <CircularProgress />;
   } else if (oscalUrl) {
-    result = props.renderer(
-      isRestMode,
-      oscalData,
-      oscalUrl,
-      onResolutionComplete,
-      handleRestPatch
+    result = isRestMode ? (
+      <Split className={classes.split} minSize={500} sizes={[50, 50]}>
+        <Box>
+          <OSCALJsonEditor value={oscalData} onSave={handleRestPut} />
```

Comment:
Here `oscalData` will often contain fields other than what's returned in the REST response due to things like profile resolution.  For example, SSPs will contain a `resolvedControls` array that should not be shown in source and should not be sent in the `PUT` REST request.

Reply:
Handled the removal of `resolvedControls` in https://github.com/EasyDynamics/oscal-react-library/pull/265/commits/cf83ad1ea976017b059d2b5f5711bda2e92a7168. If there are any others that need to be removed, please let me know. It's pretty opaque to me how those got in added to `oscalData`.

Reply:
The way the additional fields are populated and passed in the resolvers ([example](https://github.com/EasyDynamics/oscal-react-library/blob/feature/json-editor/src/components/oscal-utils/OSCALSspResolver.js#L35)) is currently a bit sloppy.

At some point we should revisit how we pass that additional info around to the components, but for now, perhaps we should consider shoving the original response string in something like `oscalData.oscalSource` rather than the approach of copying and removing what was added?